packages/utils/src/loader/cdn-loader.ts

```diff
@@ -49,39 +49,41 @@ export class CDNLoader {
    * @param href The location of the SVG to load
    * @returns Promise of the SVG body
    */
-  private async loadContent (href: string): Promise<XMLHttpRequest> {
+  private async loadContent (href: string): Promise<Response> {
     try {
-      const response = await this.xmlRequest(href);
+      const response = await this.fetchRequest(href);
       return response;
     }
     catch (e) {
       // Failed response...
       this.responseCache.delete(href);
+      if (e instanceof Response) {
+        return Promise.resolve({
+          status: e.status || 0,
+          statusText: e.statusText || 'Failed to make request'
+        } as Response);
+      }
       return Promise.resolve({
```

Comment:
It's failed, why resolve?

Reply:
loadContent is response handler. So it need to return as resolve error response to let other function takes over.